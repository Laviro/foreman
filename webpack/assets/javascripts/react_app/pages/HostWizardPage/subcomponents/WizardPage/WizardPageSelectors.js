export const selectWizardPage = state => state.wizardPage;
export const selectFilter = state => selectWizardPage(state).filter;
