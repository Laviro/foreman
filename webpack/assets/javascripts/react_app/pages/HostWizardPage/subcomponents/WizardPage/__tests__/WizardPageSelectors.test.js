import { testSelectorsSnapshotWithFixtures } from 'react-redux-test-utils';
import { selectWizardPage, selectBool } from '../WizardPageSelectors';

const state = {
  wizardPage: {
    bool: false,
  },
};

const fixtures = {
  'should return WizardPage': () => selectWizardPage(state),
  'should return WizardPage bool': () => selectBool(state),
};

describe('WizardPage selectors', () =>
  testSelectorsSnapshotWithFixtures(fixtures));
