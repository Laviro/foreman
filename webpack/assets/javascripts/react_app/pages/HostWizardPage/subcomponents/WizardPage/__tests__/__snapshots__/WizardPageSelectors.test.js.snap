// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WizardPage selectors should return WizardPage 1`] = `
Object {
  "bool": false,
}
`;

exports[`WizardPage selectors should return WizardPage bool 1`] = `false`;
